### PR TITLE
chapter5: improvements

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: manuscript/chapter5.md [hunk 1]
```diff
@@ -154,8 +154,8 @@ Now you would be able to access the input DOM element. In the example of the foc
 
 ### Exercises
 
-* read more about [the usage of the ref attribute in React](https://www.robinwieruch.de/react-ref-attribute-dom-node/)
-* read more about [the ref attribute in general in React](https://reactjs.org/docs/refs-and-the-dom.html)
+* read more about [the usage of the `ref` attribute in React](https://www.robinwieruch.de/react-ref-attribute-dom-node/)
```
Comment (rwieruch) [inline manuscript/chapter5.md]: syntax highlights cannot be used in links :)

File: manuscript/chapter5.md [hunk 2]
```diff
@@ -139,7 +139,7 @@ const Search = ({
         value={value}
         onChange={onChange}
 # leanpub-start-insert
-        ref={(node) => input = node}
+        ref={el => this.input = el}
```
Comment (rwieruch) [inline manuscript/chapter5.md:142]: Does `el` make sense for everyone? I know the React documentation uses it as well, but maybe `element` is more specific.

File: manuscript/chapter5.md [hunk 3]
```diff
@@ -154,8 +154,8 @@ Now you would be able to access the input DOM element. In the example of the foc
 
 ### Exercises
 
-* read more about [the usage of the ref attribute in React](https://www.robinwieruch.de/react-ref-attribute-dom-node/)
-* read more about [the ref attribute in general in React](https://reactjs.org/docs/refs-and-the-dom.html)
+* read more about [the usage of the `ref` attribute in React](https://www.robinwieruch.de/react-ref-attribute-dom-node/)
+* read more about [the `ref` attribute in general in React](https://reactjs.org/docs/refs-and-the-dom.html)
```
Comment (rwieruch) [inline manuscript/chapter5.md]: syntax highlight in link again

File: manuscript/chapter5.md (hunk 2 above)
Comment (lex111) [reply manuscript/chapter5.md]: Ooops, I will revert it

Comment (lex111) [reply manuscript/chapter5.md]: So with the right alignment, does it really look better than the center? https://www.fotor.com/share/92ed452d-bbe0-4ba0-b3ab-c98580061483

Comment (lex111) [reply manuscript/chapter5.md:142]: I think it's better to do so that there is no difference with the documentation?

Comment (rwieruch) [reply manuscript/chapter5.md:142]: Hm okay. Does it say "element" somewhere in the text? If not, then maybe add it over there.

Comment (lex111) [reply manuscript/chapter5.md:142]: Replace 

> The `this` object of an ES6 class component helps us to reference the DOM node with the `ref` attribute.

with

> The `this` object of an ES6 class component helps us to reference the DOM __element__ with the `ref` attribute.

Right?

Comment (rwieruch) [reply manuscript/chapter5.md:142]: Yep. Then it's alright 👍

Comment (lex111) [reply manuscript/chapter5.md:142]: Done :ballot_box_with_check:

**Pull request merged**